components/Conversations/Panel/ConversationPanel.tsx

```diff
@@ -18,16 +18,20 @@ const ConversationPanel: React.FC = () => {
 
   const { filteredConversations, isMergeActive, toggleMergeActive } = useConversations()
 
-  const last24HoursConversations = filteredConversations.filter((convo) => isLast24Hours(new Date(convo.timestamp)))
-  const last24HoursTitle = last24HoursConversations.length > 0 ? 'Last 24 Hours' : undefined
+  const TodayConversations = filteredConversations.filter((convo) => 
+    isToday(new Date(convo.timestamp))
+)
+  const TodayConversationsTitle = TodayConversations.length > 0 ? 'Today' : undefined
 
   const past7DaysConversations = filteredConversations.filter(
-    (convo) => isPast7Days(new Date(convo.timestamp)) && !isLast24Hours(new Date(convo.timestamp))
+    (convo) => isPast7Days(new Date(convo.timestamp))
   )
-  const past7DaysTitle = past7DaysConversations.length > 0 ? 'Past 7 Days' : undefined
+  const past7DaysTitle = past7DaysConversations.length > 0 ? 'This Week' : undefined
 
-  const olderConversations = filteredConversations.filter((convo) => isOlderThan7Days(new Date(convo.timestamp)))
-  const olderTitle = olderConversations.length > 0 ? 'Older' : undefined
+  const olderConversations = filteredConversations.filter(
+    (convo) => isOlderThan7Days(new Date(convo.timestamp))
+  )
+  const olderTitle = olderConversations.length > 0 ? 'Past Month' : undefined
```

Comment:
These convos aren't necessarily all within the past month - just older than a week. Going to change back to 'Older' for now so that we can merge, then you can add the "this month" logic

Reply:
Thanks for the feedback! Makes a lot of sense